File: gcse-programming-concepts/programming-constructs/simple_selection.cs
using System;

namespace isaac_code_samples
{
    class SimpleSelection
    {
        void Main(string[] args){
            Console.WriteLine("Which animal has a long bendy trunk?");
            string animal = Console.ReadLine();
            if (animal == "Elephant"){
                Console.WriteLine("You are correct");
            }else{
                Console.WriteLine("You are incorrect");
            }

        }
    }
}

File: gcse-programming-concepts/programming-constructs/simple_selection.py
def guess_the_animal():
    """Example of a simple selection statement"""
    animal = input("Which animal has a long bendy trunk? ")
    if animal == "elephant":
        print("You are correct")
    else:
        print("You are incorrect")

if __name__ == '__main__':
    guess_the_animal()
